pkg/e2e/spinup_test.go

```diff
@@ -119,7 +119,28 @@ func spinup(t testing.TB, ctx context.Context, cfg config) (chan error, error) {
 
 		cmd := c
 		g.Add(func() error {
-			err := cmd.Wait()
+			done := make(chan error)
+
+			// Lauching Wait in a goroutine and killing it if it times out.
+			// On windows it seems the executable starts successfully but Wait never returns.
+			go func() { done <- cmd.Wait() }()
+
+			timeout := time.After(5 * time.Second)
+
+			var er error
```

Comment:
nit
```suggestion
			var err error
```
then change the err in the select to something more concrete like `doneErr`

Reply:
Agreed :+1: